docs/operators/router.md

```diff
@@ -10,10 +10,11 @@ An entry that does not match any of the routes is dropped and not processed furt
 
 ### Configuration Fields
 
-| Field    | Default  | Description                              |
-| ---      | ---      | ---                                      |
-| `id`     | `router` | A unique identifier for the operator     |
-| `routes` | required | A list of routes. See below for details  |
+| Field    | Default  | Description                                                                                                                                                                                                                              |
+| ---      | ---      | ---                                                                                                                                                                                                                                      |
+| `id`     | `router` | A unique identifier for the operator                                                                                                                                                                                                     |
+| `routes` | required | A list of routes. See below for details                                                                                                                                                                                                  |
+| `if`     |          | An [expression](/docs/types/expression.md) that, when set, will be evaluated to determine whether this operator should be used for the given entry. This allows you to do easy conditional parsing without branching logic with routers. |
```

Comment:
Should we need to exclude `router` from exposing this?

It seems a little awkward to make use of it in a router, but more concerning - what happens if a router is skipped?

Reply:
Whoops. That doesn't belong there. The router doesn't embed transformer, so it's not implemented for the router